src/snowflake/cli/_plugins/snowpark/commands.py

```diff
@@ -324,6 +329,10 @@ def build(
 
     anaconda_packages_manager = AnacondaPackagesManager()
 
+    # Clean up deploy root
+    if project_paths.deploy_root.exists():
+        rmtree(project_paths.deploy_root)
+
```

Comment:
How about `project_paths.remove_up_deploy_root()` in case we use this rmtree in more places?